litmus-portal/frontend/src/components/Sections/Settings/AccountsTab/AccountSettings/index.tsx

```diff
@@ -139,99 +117,34 @@ const AccountSettings: React.FC = () => {
-                  onChange={handleNewPassword('password')}
-                  disableUnderline
-                  endAdornment={
-                    <InputAdornment position="end">
-                      <IconButton
-                        data-cy="conVisibilty"
-                        aria-label="toggle password visibility"
-                        onClick={handleClickShowPassword2}
-                        onMouseDown={handleMouseDownPassword}
-                        edge="end"
-                      >
-                        {newPassword.showPassword ? (
-                          <Visibility data-cy="visIcon" />
-                        ) : (
-                          <VisibilityOff data-cy="invisIcon" />
-                        )}
-                      </IconButton>
-                    </InputAdornment>
-                  }
-                />
-                <InputLabel htmlFor="outlined-adornment-password">
-                  New Password
-                </InputLabel>
-              </FormControl>
+              <InputField
+                required
+                type="password"
+                handleChange={handleNewPassword('password')}
+                label="New Password"
+                validationError={newPassword.err}
+                value={newPassword.password}
+              />
+
               {/* Confirm new password */}
-              <FormControl>
-                <Input
-                  data-cy="confirmPassword"
-                  className={`${classes.pass} ${
-                    confNewPassword.err ? classes.error : classes.success
-                  }`}
-                  id="outlined-adornment-password"
-                  type={confNewPassword.showPassword ? 'text' : 'password'}
-                  onChange={handleConfPassword('password')}
-                  disableUnderline
-                  endAdornment={
-                    <InputAdornment position="end">
-                      <IconButton
-                        data-cy="conVisibilty"
-                        aria-label="toggle password visibility"
-                        onClick={handleClickShowPassword3}
-                        onMouseDown={handleMouseDownPassword}
-                        edge="end"
-                      >
-                        {confNewPassword.showPassword ? (
-                          <Visibility data-cy="visIcon" />
-                        ) : (
-                          <VisibilityOff data-cy="invisIcon" />
-                        )}
-                      </IconButton>
-                    </InputAdornment>
-                  }
-                />
-                <InputLabel htmlFor="outlined-adornment-password">
-                  Confirm new password
-                </InputLabel>
-              </FormControl>
+              <InputField
+                required
+                type="password"
+                handleChange={handleConfPassword('password')}
```

Comment:
Add a `success` field here, you can use validateConfirmPassword and negate it or use useRef. Take a look at `Stepper` password validation for reference.